lib/Form/DisplayFormInWPContent.php

```diff
@@ -182,14 +186,21 @@ private function shouldDisplayFormType(FormEntity $form, string $formType): bool
     if (!is_array($settings)) return false;
 
     $keys = self::SETUP[$formType];
+
+    $switchKey = $keys['switch'];
+    if (isset($settings[$switchKey]) && ($settings[$switchKey] !== '1')) {
```

Comment:
I think it might be better to use `!isset($settings[$switchKey]) || ($settings[$switchKey] !== '1'` so that forms without the property are disabled as well.